packages/x-scheduler/src/material/internals/components/event-popover/EventPopover.test.tsx

```diff
@@ -1,26 +1,36 @@
 import * as React from 'react';
-import { DateTime } from 'luxon';
 import { screen } from '@mui/internal-test-utils';
 import { createSchedulerRenderer } from 'test/utils/scheduler';
 import { CalendarEvent, CalendarResource } from '@mui/x-scheduler/primitives/models';
 import { StandaloneView } from '@mui/x-scheduler/material/standalone-view';
 import { spy } from 'sinon';
 import { Popover } from '@base-ui-components/react/popover';
+import { getAdapter } from '../../../../primitives/utils/adapter/getAdapter';
 import { EventPopover } from './EventPopover';
 
+const adapter = getAdapter();
+
 const calendarEvent: CalendarEvent = {
   id: '1',
-  start: DateTime.fromISO('2025-05-26T07:30:00'),
-  end: DateTime.fromISO('2025-05-26T08:15:00'),
+  start: adapter.date('2025-05-26T07:30:00'),
+  end: adapter.date('2025-05-26T08:15:00'),
   title: 'Running',
   description: 'Morning run',
+  resource: 'r2',
 };
 
-const calendarEventResource: CalendarResource = {
-  id: 'r1',
-  name: 'Personal',
-  eventColor: 'cyan',
-};
+const resources: CalendarResource[] = [
```

Comment:
Could you add a test the checks that the form behave correctly when the resource has no color associated?